palantirnet/st-bernards
# Contentimport invalid json; drush aliases

## JIRA Ticket(s)

- See ticket [MSSB-96: February Deployment 1](https://palantir.atlassian.net/browse/MSSB-96).

## Description

- Adds a patch for the Content Import module to fix invalid JSON in `composer.json` (see https://www.drupal.org/project/contentimport/issues/2959786)
- Adds Drush aliases for Pantheon environments

## Testing instructions

- `vagrant ssh`
- `drush sa`
- Notice the new Drush aliases
- `drush @pantheon.develop status`
- Verify Drush successfully runs against the `develop` multidev on Pantheon.
- Verify you no longer see `>  [warning] Invalid json in modules/contentimport/composer.json` when running Drush commands.
